package.json

```diff
@@ -135,7 +135,7 @@
     "bpk-component-tooltip": "4.0.21-css.0",
     "bpk-mixins": "19.0.21-css.0",
     "bpk-react-utils": "2.8.3-css.0",
-    "bpk-stylesheets": "4.1.2-css.0",
+    "bpk-stylesheets": "^4.1.4-css.0",
```

Comment:
Same as above

Reply:
A true, I tried to force that, not sure what happened